egs/multi_lang/s5/run-2c-bnf-v3.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+# -v3 is as -v2 but it just consists 10 closest fullLP langs to GRG + GRG.
+# -v2 is as 2c-bnf.sh but it consists of FullLP GRG.
+
+# Copyright 2016  Pegah Ghahremani
+# Apache 2.0
+
+#This yields approx 70 hours of data
+# this script generates bottleneck features from multilingual model
+# trained on list of languages and dump the bnf for specific language L.
+set -e           #Exit on non-zero return code from any command
+set -o pipefail  #Exit if any of the commands in the pipeline will
+                 #return non-zero return code
+. conf/common_vars.sh || exit 1;
+
+set -u           #Fail on an undefined variable
+skip_kws=true
+skip_stt=false
+bnf_train_stage=-100
+stage=1
+num_archives=20
+bnf_init_lrate=0.0017
+bnf_final_lrate=0.00017
```

Comment:
common.limitedLP has all these parameters. Please reuse them.
You have currently made a softlink of the common.limitedLP. Instead you can make a copy and modify the parameters there.